author.bib

```diff
@@ -326,11 +341,13 @@ @string{ Gaudiano_P
 @string{ Gautrais_J           = "Gautrais, Jacques" }
 @string{ Gelatt_CDJr          = "Gelatt, Jr., C. Daniel" }
 @string{ Geman_S              = "Geman, Stuart" }
+@string{ Gent_IP              = "Gent, Ian P." }
```

Comment:
Should appear after Gendreau_M (Ganguli_M is misplaced; should appear after Gambin_T)